src/vmm/src/vmm_config/drive.rs

```diff
@@ -70,109 +81,114 @@ pub struct BlockDeviceConfig {
     pub rate_limiter: Option<RateLimiterConfig>,
 }
 
-/// Wrapper for the collection that holds all the Block Devices Configs
+/// Wrapper for the collection that holds all the Block Devices
 #[derive(Default)]
-pub struct BlockDeviceConfigs {
-    /// A list of `BlockDeviceConfig` objects.
-    pub config_list: VecDeque<BlockDeviceConfig>,
-    has_root_block: bool,
+pub struct BlockDevices {
```

Comment:
Is it possible to have a generic struct definition/interface for Vsock/Block/Net stores/builders ?